tests/unit_tests/simulator/test_simulation_context.py

```diff
@@ -62,11 +61,7 @@ def test_simulation_context(setup_case, storage, monkeypatch, try_queue_and_sche
             assert even_ctx.didRealizationSucceed(iens)
             assert not even_ctx.didRealizationFail(iens)
             assert even_ctx.isRealizationFinished(iens)
-
-            assert even_half.state_map[iens] == RealizationStorageState.HAS_DATA
         else:
             assert odd_ctx.didRealizationSucceed(iens)
             assert not odd_ctx.didRealizationFail(iens)
             assert odd_ctx.isRealizationFinished(iens)
-
-            assert odd_half.state_map[iens] == RealizationStorageState.HAS_DATA
```

Comment:
Add a different assert here or OK to remove?

Reply:
As above